modules/luci-base/htdocs/luci-static/resources/form.js

```diff
@@ -3609,6 +3629,7 @@ var CBIFlagValue = CBIValue.extend(/** @lends LuCI.form.FlagValue.prototype */ {
 			value_enabled: this.enabled,
 			value_disabled: this.disabled,
 			validate: L.bind(this.validate, this, section_id),
+			tooltip: this.tooltip ? L.bind(this.tooltip, this, section_id) : null,
```

Comment:
Since you introduce the new `tooltip` property in the `LuCI.form.AbstractElement` properties, you should add support for it to all widget renderers and not just flag values. Alternatively move the default declaration & JSDoc comment above this function to the other `LuCI.form.FlagValue` specific properties.